specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/examples/RedisCacheForceReboot.json

```diff
@@ -6,11 +6,7 @@
     "subscriptionId": "subid",
     "parameters": {
       "shardId": 0,
-      "rebootType": "AllNodes",
-      "ports": [
-        13000,
-        15001
-      ]
+      "rebootType": "AllNodes"
```

Comment:
Was this change because really you have to choose either rebootType or ports? If so makes sense.

Reply:
it will raise anoher error: `(InvalidRequestBody) The value of the parameter 'Ports' is invalid.`